modules/core/nuxeo-core-binarymanager-cloud/nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/core/storage/sql/S3DirectBatchHandler.java

```diff
@@ -339,6 +334,19 @@ public boolean completeUpload(String batchId, String fileIndex, BatchFileInfo fi
         return true;
     }
 
+    protected String doMove(CopyObjectRequest request) {
```

Comment:
Could you add a Javadoc explaining that it returns the ETag of the resulting blob?